gateway/app/settings.py

```diff
@@ -23,6 +23,11 @@ class OTPBackend(Enum):
     Redis = "redis"
 
 
+class QodProvisioningBackend(str, Enum):
+    Mock = "mock"
```

Comment:
If you add a variant to the enum you must implement it, but you can remove this for now.